lib/PuppeteerSharp.Tests/PageTests/PdfTests.cs

```diff
@@ -143,27 +145,5 @@ public void PdfOptionsShouldBeSerializable()
             var newPdfOptions = JsonSerializer.Deserialize<PdfOptions>(serialized);
             Assert.That(newPdfOptions, Is.EqualTo(pdfOptions));
         }
-
-        [Test]
-        public void PdfOptionsShouldWorkWithMarginWithNoUnits()
```

Comment:
This was testing something that was causing puppeteer to fail. Let's leave it so we prove that we can pass `"0"`.

Reply:
I added i back. I am still not sure that it actually testes the `ConvertPrintParameterToInches` code path.

https://github.com/hardkoded/puppeteer-sharp/blob/cbff8a1d6eee3f2b32fced57406d38ac941fb88a/lib/PuppeteerSharp/Cdp/CdpPage.cs#L1217-L1257

Reply:
That test should check [this change](https://github.com/hardkoded/puppeteer-sharp/commit/a4ce0cfab1d376bcb84c534f24a1762cd02db802#diff-b7e03018805bd7ee45bb7f4eec5eed50f785de53ec455aaa07a896af2c259a7aR1231)

Reply:
You're right that test is wrong.

Reply:
I turned that test into a (valid) more unit test. Let me know what you think.